src/fobnail-attester.c

```diff
@@ -10,13 +10,18 @@
 #include <signal.h>
 #include <stdbool.h>
 
+#include <meta.h>
```

Comment:
@0xDen This may conflict with other headers, e.g. if for any reason there is a `meta.h` file in `/usr/include` etc.
File should be included like that
```c
#include <fobnail-attester/meta.h>
```

Reply:
Fixed